### DISCUSSION TITLE
Need to merge accounts together

### DISCUSSION THREAD

**Discussion by ghost** [community/community#73654] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I had an original Github account when I was in college. It was linked to my school email...recently my college decided to mass delete all school emails so I can no longer get into my GitHub account to change the email address because I can no longer verify it's me (the verification email is going to an email that no long exists). I need to change the email on that account to my new personal email and I have no idea how to do it.

**Comment by Naman-B-Parlecha** [comment]
wait so u are login into ur clg github acc??

**Reply by Naman-B-Parlecha** [reply]
if yes then it is easy just to setting -> emails -> add email -> delete the clg email id and the ur new email will become primary email

if no then it is very hard and u will have to reach out to github support team and solve the issue with them

**Reply by ghost** [reply]
Yes. I'm a recent college graduate and want my old work for reference on my resume and going back to look at and refine in the future!

**Reply by Naman-B-Parlecha** [reply]
if u find this helpful then please mark this as th answer and for more doubt just tag me 
thank you

**Reply by ghost** [reply]
I can't get into the github linked to my college email because I can't verify it's me. The verification email is sent to an email that no longer exists

**Reply by ghost** [reply]
how can I get in touch with the Github support team?

**Reply by Naman-B-Parlecha** [reply]
Go to the GitHub Support page: https://support.github.com/

**Reply by Naman-B-Parlecha** [reply]
can u mark this as the answer @ashleyweavie thanks

**Reply by Naman-B-Parlecha** [reply]
wait did u do 2FA??

**Comment by OmBharsakle** [comment]
hi help me
